stock_measuring_device_zippcube/controllers/main.py

```diff
@@ -10,26 +10,39 @@
 _logger = logging.getLogger(__name__)
 
 
-class ZipcubeController(http.Controller):
+class ZippcubeController(http.Controller):
     weight_keys = ("weight",)
     measures_keys = ("length", "width", "height")
     expected_keys = ("secret", "barcode") + weight_keys + measures_keys
 
-    @http.route("/stock/zipcube/<int:id_>/measurement", type="json", auth="none")
-    def measurement(self, id_):
-        env = request.env(su=True)
-        cubiscan = env["cubiscan.device"].browse(id_)
-        if not cubiscan:
-            raise MissingError(_("No such cubiscan"))
+    @http.route(
+        "/stock/zippcube/<string:zippcube_device_name>/measurement",
```

Comment:
why not `<model(zippcube.device):device>`? You'll have the object straight using its slug.

Reply:
I don't find any documentation on how this works. Does it search by the model's name?